src/EA.Iws.Web/Views/NewNotification/NotificationType.cshtml

```diff
@@ -6,6 +7,21 @@
     ViewBag.Title = NotificationTypeResources.Title;
 }
 
+@helper ContactInfo(UKCompetentAuthority competentAuthority)
+{
+    @EA.Prsd.Core.Helpers.EnumHelper.GetDescription(competentAuthority)
+    @: - Telephone: 
+    switch (competentAuthority)
+    {
+        case UKCompetentAuthority.Wales:
+            @:03000 65 3000
+            break;
+        default:
+            @Html.CompetentAuthorityHelpline(competentAuthority)
+            break;
+    }
```

Comment:
For some reason NRW have a different number than their helpline for disposals. Still waiting to find out if this is correct.